deployment/legacy-migration/maintenance-worker/migration/neo4j/contributions/contributions.cql

```diff
@@ -148,6 +148,10 @@ MATCH (c:Category {id: categoryId})
 MERGE (p)-[:CATEGORIZED]->(c)
 WITH p, post.tags AS tags
 UNWIND tags AS tag
-MERGE (t:Tag {id: apoc.text.clean(tag), disabled: false, deleted: false})
+WITH apoc.text.replace(tag, '[^\\p{L}0-9]', '') as tagNoSpacesAllowed
+CALL apoc.when(tagNoSpacesAllowed =~ '^((\\p{L}+[\\p{L}0-9]*)|([0-9]+\\p{L}+[\\p{L}0-9]*))$', 'RETURN tagNoSpacesAllowed', '', {tagNoSpacesAllowed: tagNoSpacesAllowed})
```

Comment:
Could you put these expressions in a variable and give them a human readable name? `stringConsistingOfNumbersOnly` might not be short and sweet but gives me an idea of what's happening here. :wink:

Reply:
Yes, a good improvement! 🤗😍
@mattwr18 did already … 😍

… he gave an other name  …
… and I changed the name again for clarification.